.github/workflows/ci.yml

```diff
@@ -61,6 +61,7 @@ jobs:
         env:
           EXCHANGE_API_BASE_URI: http://api.exchangeratesapi.io/v1
           EXCHANGE_API_KEY: ${{ secrets.EXCHANGE_API_KEY }}
+          ALERTS_PROVIDER_SIGNING_KEY: ${{ secrets.ALERTS_PROVIDER_SIGNING_KEY }}
```

Comment:
Is it possible to generate the signing key on the test itself? That would mean that we wouldn't need to configure it in the actions.

Reply:
I've moved this to `e2e-setup` in a57a546ec1e5824d6de4f58f09b739af118f2211.